pkg/filesystem/subvolume.go

```
@@ -52,6 +60,48 @@ func List(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace
 	listCephFSSubvolumes(ctx, clientsets, operatorNamespace, clusterNamespace, includeStaleOnly, subvolumeNames)
 }
 
+// checkForExternalStorage checks if the external mode is enabled.
+func checkForExternalStorage(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace string) bool {
+	enable := false
+	cephclusters, _ := clientsets.Rook.CephV1().CephClusters(clusterNamespace).List(ctx, v1.ListOptions{})
```

why err is discarded here?